packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react-forms.test.ts

```diff
@@ -228,7 +228,7 @@ describe('amplify form renderer tests', () => {
       expect(componentText).toContain('const studentRecords = useDataStoreBinding({');
 
       // check lazy load linked data
-      expect(componentText).toContain('const linkedStudents = record ? await record.Students.toArray() : [];');
+      expect(componentText).toContain('(record && (await record.Students?.toArray())) || [];');
```

Comment:
This line has grown long enough that it is split into 2 lines that is why we no longer see `const linkedStudents =` as part of the expect clause, but it is still in the generated code just on the line above this one.